snuba/clickhouse/native.py

```diff
@@ -168,7 +168,7 @@ def execute_robust(
             except (errors.NetworkError, errors.SocketTimeoutError, EOFError) as e:
                 # Try 3 times on connection issues.
                 logger.warning(
-                    "Write to ClickHouse failed: %s (%d tries left)",
+                    "Robust ClickHouse query execution failed: %s (%d tries left)",
```

Comment:
I would drop "robust" from the message, but I'd rather just get this merged than slow you down